notebook/static/notebook/js/notebook.js

```diff
@@ -1446,6 +1446,12 @@ define(function (require) {
         if (indices.length <= 1) {
             return;
         }
+
+        // Check if trying to merge above on topmost cell
+        if (indices.indexOf(-1) > -1) {
```

Comment:
Hm. Yes. I wasn't anticipating any circumstances where an index value less
than negative one would be passed but I agree that adding a check for all
negative numbers would be better.

I'll add this as soon as I'm near a computer.
On Aug 26, 2015 7:28 AM, "Matthias Bussonnier" <notifications@github.com>
wrote:

> In notebook/static/notebook/js/notebook.js
> <https://github.com/jupyter/notebook/pull/340#discussion_r37988111>:
>
> > @@ -1446,6 +1446,12 @@ define(function (require) {
> >          if (indices.length <= 1) {
> >              return;
> >          }
> > +
> > +        // Check if trying to merge above on topmost cell
> > +        if (indices.indexOf(-1) > -1) {
>
> Do we want to be more restrictive and check that no indices is negative ? (.filter(function(value){return
> value < 0}))
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/notebook/pull/340/files#r37988111>.
>